README.md

```diff
@@ -11,6 +11,7 @@ It automates the process of logging into clusters via Teleport (`tsh`) and setti
 *   **Simplified Connection:** Connect to management and workload clusters with a single command.
 *   **Automatic Context Switching:** Sets your `kubectl` context correctly.
 *   **Prometheus Port-Forwarding:** Handles `kubectl port-forward` for the Mimir query frontend automatically.
```

Comment:
The README should be updated to reflect that Grafana is also port-forwarded. 

Suggestion:

Change:
`*   **Prometheus Port-Forwarding:** Handles `kubectl port-forward` for the Mimir query frontend automatically.`

To:
`*   **Service Port-Forwarding:** Automatically handles `kubectl port-forward` for services like Prometheus (Mimir query frontend) and Grafana on the Management Cluster, and Alloy Metrics on the Workload Cluster.`

Consider also adding a note under "MCP Integration Notes" if Grafana is relevant there, or just its availability e.g., `http://localhost:3000`.